torch/nn/functional.py

```diff
@@ -841,12 +849,6 @@ def max_pool3d_with_indices(
         return_indices: If ``True``, will return the argmax along with the max values.
                         Useful for :class:`torch.nn.functional.max_unpool3d` later
     """
-    # See: https://github.com/pytorch/pytorch/pull/62544#issuecomment-896195121
-    # and https://github.com/pytorch/pytorch/issues/62545 for context
-    if ceil_mode != return_indices:
-        warnings.warn("Note that order of the arguments: ceil_mode and return_indices will change"
-                      "to match the args list in nn.MaxPool3d in a future release.")
-
```

Comment:
If these warnings are removed, the error for `Proxy object cannot be iterated` is not called. These test failures from `test/test_fx.py` will also be raised in the master branch if this branch of `if` is removed. Not sure of the solution, but the failures aren't directly related to the changes in this PR. Please correct me if I'm wrong.

cc: @jbschlosser

Reply:
@krshrimali You can fix the FX tests by editing the lines here:
https://github.com/pytorch/pytorch/blob/6c8e516a8034d7ab615025ac57b5bc2e7680efb1/test/test_fx.py#L3672-L3674

Changing `CONTROL_FLOW` -> `ARG_TYPE_MISMATCH` should do the trick. If not, there's a set of error messages the test is looking for here:
https://github.com/pytorch/pytorch/blob/6c8e516a8034d7ab615025ac57b5bc2e7680efb1/test/test_fx.py#L3566-L3573

Reply:
Thanks, @jbschlosser for your response and help! Please allow me a couple of days to get back to you and update the PR, if that's fine...